### ISSUE TITLE
Issue with Login - Deprecated

### ISSUE THREAD

**Issue by zane-shus** [mwgg/iracing-php#2]
I have basic code:

```
require_once __DIR__ . '/vendor/autoload.php';
	use iRacingPHP\iRacing;
	$iracing = new iRacing('zane@outlook.com', 'mypassword');
```

Its throwing this error:

```
Deprecated: Creation of dynamic property iRacingPHP\iRacing::$username is deprecated in /var/www/html/vendor/mwgg/iracing-php/src/iRacing.php on line 13

Deprecated: Creation of dynamic property iRacingPHP\iRacing::$iRacingPHP\Data\Member is deprecated in /var/www/html/vendor/mwgg/iracing-php/src/iRacing.php on line 28
```

In case it matters, a php-v gives this
```
php -v
PHP 8.2.0 (cli) (built: Dec  8 2022 19:39:31) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.2.0, Copyright (c) Zend Technologies
```

From my composer:
`"mwgg/iracing-php": "^1.3"`

Any thoughts?

**Comment by mwgg** [comment]
Looks like it has to do with changes introduced with PHP 8.2 that deprecate dynamic properties. Looking back at my code, I have no idea why I'm creating dynamic properties anyway.

I'll get it fixed next week, likely on Monday. In the meantime you can roll back to 8.1 or earlier.

**Comment by mwgg** [comment]
Fixed with 6f4695b4829b9aa57a18ca4d4c2238e0c6186f4c (v1.4.0).

**Comment by zane-shus** [comment]
this great, thank you.
